lib/segment/src/types.rs

```diff
@@ -355,7 +355,7 @@ pub struct ScalarQuantizationConfig {
     pub r#type: ScalarType,
     /// Quantile for quantization. Expected value range in (0, 1.0]. If not set - use the whole range of values
```

Comment:
```suggestion
    /// Quantile for quantization. Expected value range in (0.5, 1.0]. If not set - use the whole range of values
```

Reply:
Updated! Changed it to `[0.5, 1.0]` instead because the validation range is inclusive on both sides.